lib/models/message.dart

```diff
@@ -1,20 +1,26 @@
-import './model.dart';
+import 'package:chirp_nets/models/model.dart';
 
 class Message implements Model {
   final int id;
+  final int createdBy;
+  final int conversationId;
```

Comment:
id vs conversationId?

Reply:
same here id is the message unique identifier and the conversationId is the conversations unique identifier